src/DIPS.Xamarin.UI/Controls/Popup/PopupBehavior.cs

```diff
@@ -74,6 +74,21 @@ public PopupDirection Direction
             set { SetValue(DirectionProperty, value); }
         }
 
+        /// <summary>
+        /// <see cref="Animation" />
+        /// </summary>
+        public static readonly BindableProperty AnimationProperty =
+            BindableProperty.Create(nameof(Animation), typeof(PopupAnimation), typeof(PopupBehavior), PopupAnimation.None);
+
+        /// <summary>
+        /// How the popup will animate into view. Either none, sliding or fading.
+        /// </summary>
+        public PopupAnimation Animation
```

Comment:
This property does not need to be a `BindableProperty`. It is handy that it is, but this is most likely something our consumers are going to set once (in their XAML) and not change with using Binding.